tests/non_silent.rs

```diff
@@ -0,0 +1,48 @@
+// pest. Elegant, efficient grammars
+// Copyright (C) 2016  Dragoș Tiselice
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#[macro_use]
+extern crate pest;
+
+use pest::prelude::*;
+
+impl_rdp! {
+    grammar! {
+        abc = { ["a"] ~ ["b"] | ["a c"] }
+
+        whitespace = { [" "] }
```

Comment:
@Keats Whitespace will work just as it did until now, without automatically being matched before rules. So you're `if_tag` rule can have whitespace between sub-rules. If you don't want whitespace between sub-rules, you use atomic.